editor/project_converter_3_to_4.cpp

```diff
@@ -2729,6 +2737,22 @@ Vector<String> ProjectConverter3To4::check_for_rename_common(const char *array[]
 	return found_renames;
 }
 
+// Convert hexadecimal colors from ARGB to RGBA
+void ProjectConverter3To4::convert_hexadecimal_colors(Vector<SourceLine> &source_lines, const RegExContainer &reg_container, bool builtin) {
```

Comment:
```suggestion
void ProjectConverter3To4::convert_hexadecimal_colors(Vector<SourceLine> &source_lines, const RegExContainer &reg_container) {
```

It doesn't do anything?

Reply:
Method should also go next to `ProjectConverter3To4::rename_colors` in both the header and the cpp file.